.drone.yml

```diff
@@ -100,5 +108,15 @@ pipeline:
       status: [ failure ]
       event: [ push, tag, deployment ]
 
+  rebuild-cache:
+    image: drillster/drone-volume-cache
+    rebuild: true
+    mount:
+      - /drone/cache/ivy2
+    volumes:
+      - /tmp/cache:/cache
+#    when:
+#     event: push
```

Comment:
What is the default?

Reply:
Everything: [push, pull_request, tag, deployment]. I think we should switch to push only.
I commented it out to make sure I can build a cache and reuse a cache